packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -513,9 +521,9 @@ export abstract class BaseSegment implements ISegment {
 		seq?: number,
 		collaborating?: boolean,
 		rollback: PropertiesRollback = PropertiesRollback.None,
-	) {
+	): PropertySet {
 		this.propertyManager ??= new PropertiesManager();
-		// eslint-disable-next-line import/no-deprecated
+		// eslint-disable-next-line import/no-deprecated, @typescript-eslint/no-explicit-any
```

Comment:
Can we leave a comment explaining why we can't move away from `any`? Or leave a TODO to come back and clean this up?